packages/datx-codemod/prepublish.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+
+const packageJson = require('./package.json');
+
+delete packageJson.scripts;
+delete packageJson.devDependencies;
+delete packageJson.jest;
+delete packageJson.husky;
+
+const distPackageJson = {
+  ...packageJson,
+  bin: './bin/datx-codemod.js',
+};
+
+fs.writeFileSync('./dist/package.json', JSON.stringify(distPackageJson, null, 2));
+
+fs.copyFileSync('./README.md', './dist/README.md');
+fs.copyFileSync('./LICENSE', './dist/LICENSE');
+
+fs.rmdirSync('./dist/transforms/__testfixtures__', { recursive: true });
```

Comment:
What should I add to it?
Should I copy `.npmignore` to `dist` folder then? 
Because we are doing release from dist folder.
We are not doing this in any other package.

Reply:
yeah, I think it should be in the folder that is beeing published. By default, npm is taking gitignore if npmignore doesn't exist, so that's why we didn't really need it until now.

Reply:
Is this what you meant?
![image](https://user-images.githubusercontent.com/4128883/236855207-458a7c92-4f41-44d0-a2ef-d5de220903c5.png)

Honestly, I don't like this setup.
It would make sense if we are not publishing from the dist folder, e.g. if we publish from the root and we don't want to publish some dev stuff. 

We need to rethink publishing process and make a setup that would allow publishing form the package root folder, otherwise it might be confusing.

Reply:
https://www.npmjs.com/package/clean-package this tool can clean the root package.json form all the nonsense before publish and revert it back.
Then we will be able to publish form the root folder and in that case `.npmignore` would make complete sense.

Reply:
Ok, leave it as is for now